style.css

```diff
@@ -0,0 +1,84 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  width: 100%;
+}
+
+ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  gap: 6vw;
+}
+
+hr {
+  width: 95%;
+  border: 1px solid #212121;
+}
+
+/* header  */
+header {
+  background: #3c3a39;
+  width: 100%;
+  height: 11.2vh;
+  display: flex;
+}
+
+header h1 {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  display: flex;
+  gap: 200px;
+  align-items: center;
+  margin-left: 15px;
+}
+
+header img {
+  padding-left: 57vw;
+}
+
+/* intro */
+#intro {
+  background: #1c1a19;
+  width: 100%;
+  background-image: url("./img/intro-mobile.png");
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  height: 167vh;
```

Comment:
- [ ] I recommend using `px` units instead of `vh` units to fix the size of the home page as requested by the previous code reviewer to be similar to the Figma design.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/226700180-0569535e-7df0-4a1f-aa07-9c59b135d57b.png) | ![image](https://user-images.githubusercontent.com/58642949/226700221-5ca28955-77b6-403e-bb63-6715875830e9.png) |